src/content/extensions/multipleroutes/src/options.js

```diff
@@ -1,97 +1,114 @@
 'use strict';
 
+var pn = chrome.privacy.network;
+var pi = chrome.privacy.IPHandlingPolicy;
+
+var mapPolicyToRadioId = {};
+mapPolicyToRadioId[pi.DEFAULT] = 0;
+mapPolicyToRadioId[pi.DEFAULT_PUBLIC_AND_PRIVATE_INTERFACES] = 1;
+mapPolicyToRadioId[pi.DEFAULT_PUBLIC_INTERFACE_ONLY] = 2;
+mapPolicyToRadioId[pi.DISABLE_NON_PROXIED_UDP] = 3;
+
+var mapRadioIdToPolicy = {};
+if (pn.webRTCIPHandlingPolicy === undefined) {
+  // radio id => [|webRTCMultipleRoutesEnabled|, |webRTCNonProxiedUdpEnabled|]
+  // The [1] option won't exist if pn.webRTCIPHandlingPolicy is undefined.
+  mapRadioIdToPolicy[0] = [true, true];
+  mapRadioIdToPolicy[2] = [false, true];
+  mapRadioIdToPolicy[3] = [false, false];
+} else {
+  mapRadioIdToPolicy[0] = pi.DEFAULT;
+  mapRadioIdToPolicy[1] = pi.DEFAULT_PUBLIC_AND_PRIVATE_INTERFACES;
+  mapRadioIdToPolicy[2] = pi.DEFAULT_PUBLIC_INTERFACE_ONLY;
+  mapRadioIdToPolicy[3] = pi.DISABLE_NON_PROXIED_UDP;
+}
+
 // Saves options.
 function saveOptions() {
-  var multiRoutes;
-  var nonProxiedUdp;
-  var radios = document.getElementsByName('routeselection');
-  for (var i = 0, length = radios.length; i < length; i++) {
+  var radios = document.getElementsByName('ip_policy_selection');
+  var i = 0;
+  for (; i < radios.length; i++) {
     if (radios[i].checked) {
-      // option 0: multiple routes enabled, non proxied udp enabled.
-      // option 1: multiple routes disabled, non proxied udp enabled.
-      // option 2: both are disabled.
-      multiRoutes = i < 1;
-      nonProxiedUdp = i < 2;
       break;
     }
   }
 
-  chrome.privacy.network.webRTCMultipleRoutesEnabled.set({
-    'value': multiRoutes
-  });
-  try {
-    chrome.privacy.network.webRTCNonProxiedUdpEnabled.set({
-      'value': nonProxiedUdp
+  if (pn.webRTCIPHandlingPolicy !== undefined) {
+    pn.webRTCIPHandlingPolicy.set({
+      value: mapRadioIdToPolicy[i]
+    });
+  } else {
+    var mapping = mapRadioIdToPolicy[i];
+    pn.webRTCMultipleRoutesEnabled.set({
```

Comment:
Might be more readable as:

var oldBools = mapRadioIdToPolicy[i];
...
value: oldBools.allowMultipleRoutes
value: oldBools.allowUdp

Then you need to change your [true, false], etc above into {allowMultipleRoutes: true, allowUdp: false}, but that would be more readable, I think.